Source/FullScreenPokemon.js

```diff
@@ -6486,14 +6493,22 @@ var FullScreenPokemon;
             }
         };
         /**
+         * Scrolls the game window vertically
```

Comment:
You seem to limit the width of your lines pretty strictly. There's no real convention in FSS, but can you aim for ~70 on the ones in this PR? Seems weird to have three lines for these and others.